code/components/gta-streaming-five/include/EntitySystem.h

```diff
@@ -5,7 +5,7 @@
 
 #include <Pool.h>
 
-#include <boost/type_index/ctti_type_index.hpp>
+#include <boost/type_index/ctti_type_index.hpp>

```

Comment:
What happened to this line?

Reply:
Hmm not sure, maybe some other new line characters, just throwing up something here